src/app/components/category-card/category-card.component.html

```diff
@@ -0,0 +1,91 @@
+<div class="card" style="margin: auto">
+    <form [formGroup]="categoryForm">
+      <div class="loading-indicator" *ngIf="loading === true">
+        <mat-progress-spinner
+          mode="indeterminate"
+          color="accent"
+        ></mat-progress-spinner>
+      </div>
+      <div class="cat-Action">
```

Comment:
"cat-action"? Not a very self describing class name. I think "action-container", or sth similar, sounds better